### PR TITLE
Adding Docker Unit Testing

### PR DESCRIPTION
**Pull request by bobstrecansky**
I wrote a docker wrapper for performing unit tests for this repository.  
Usage can be found in the README.md

@nekufa - I wasn't sure if you were planning on using any of the code coverage drivers; when I initially pulled this down it wasn't working on my local machine but I wanted to valiate with you.  Note about that inline.

Output from the built docker container:

```
docker run -it opentelemetry-php-unittests
PHPUnit 7.5.16 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.23
Configuration: /usr/src/myapp/phpunit.xml.dist
Error:         No code coverage driver is available

.........                                                           9 / 9 (100%)

Time: 27 ms, Memory: 4.00 MB

OK (9 tests, 55 assertions)
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "composer/xdebug-handler": "^1.3"
```
Comment (bobstrecansky) [inline composer.json]: This won't be necessary if we don't want to use a code coverage driver.  Happy to remove it.

Comment (nekufa) [reply composer.json]: i suggest move it to require-dev section, it is not needed for production environments

**Comment by nekufa** [comment]
Hello, @bobstrecansky, thanks for your contribution!

File: composer.json (hunk 1 above)
Comment (bobstrecansky) [reply composer.json]: Completed.

**Comment by nekufa** [comment]
@bobstrecansky contact me https://telegram.me/nekufa

**Pull request merged**